core-api/src/main/java/com/sequenceiq/cloudbreak/api/model/stack/cluster/ClusterResponse.java

```diff
@@ -80,7 +81,10 @@
     private String ambariServerUrl;
 
     @ApiModelProperty(StackModelDescription.USERNAME)
-    private String userName;
+    private SecretResponse userName;
```

Comment:
We might need to return the CB username and password as well for smartsense. @akanto @lnardai do you guys know how will it be used?

Reply:
@mhmxs I spoke with Attila we don't need this at all nor the cb user/pass.